openhantek/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ file(GLOB_RECURSE SRC "src/*.cpp")
 file(GLOB_RECURSE HEADERS "src/*.h")
 file(GLOB_RECURSE QRC "res/*.qrc")
 
-add_custom_target(format SOURCES ".clang-format"
```

Comment:
I'd rather prefer the clang format file to me moved (and to provide an absolute path here). Otherwise the file will not appear in the project view of qtcreator and visual studio.